src/resolver.rs

```diff
@@ -2506,37 +2506,63 @@ impl<'a> ResolveReport<'a> {
                 .push(s);
         }
 
-        let mut last_suggest = vec![];
-        for suggestion in &suggestions {
-            let package = self.graph.nodes[suggestion.package].name.to_string();
-            let command = if suggestion.suggested_diff.from == ROOT_VERSION {
-                FetchCommand::Inspect {
-                    package,
-                    version: suggestion.suggested_diff.to.clone(),
-                }
-            } else {
-                FetchCommand::Diff {
-                    package,
-                    version1: suggestion.suggested_diff.from.clone(),
-                    version2: suggestion.suggested_diff.to.clone(),
-                }
-            };
-            let criteria = self
-                .criteria_mapper
-                .all_criteria_names(&suggestion.suggested_criteria)
-                .map(CriteriaName::from)
-                .collect::<Vec<_>>();
-            last_suggest.push(SuggestedAudit { command, criteria });
-        }
-        cache.set_last_suggest(last_suggest);
-
         Ok(Some(Suggest {
             suggestions,
             suggestions_by_criteria,
             total_lines,
         }))
     }
 
+    /// Given a package name and a delta to be certified, determine the set of
+    /// additional criteria for that delta/version pair which would have a
+    /// healing impact on the audit graph.
+    ///
+    /// This is more reliable than running a suggest and looking for a matching
+    /// output, as it will also select criteria for non-suggested audits.
```

Comment:
Interesting, I think I agree. Hooray for computing very precise results before even knowing what our actual goal is 🎉